### DISCUSSION TITLE
Raspberry Pi - Octoprint

### DISCUSSION THREAD

**Discussion by proEngineering** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1667] [Q&A]
Hello, people,

I have this Bigtreetech TFT70 (https://smile.amazon.de/BIGTREETECH-Touchscreen-Display-Controller-Steuerplatine-3D-Druckerteil/dp/B08B187CQ6/ref=sr_1_2? __ mk_de_DE = % C3%85M%C3%85%C5%BD%C3%95%C3%91&dchild=1&keywords=bigtreetech+tft70&qid=1614942363&sr=8-2) bought.

I control my printer with a Raspberry Pi and Octoprint (due to camera and cura to send directly from PC).

Now I noticed, however, that I cannot cancel a running print via the touch mode. I'd have to change to Marlin Mode for that.

Is there any way to realize this?

**Comment by guruathwal** [comment]
you can only cancel a print if you are printing a file from the Onboard SD card (Sd card on the mainboard). If you are printing a file from Octoprint, then it can only be cancelled from Octoprint.
